ansible/group_vars/all/config.yml

```
@@ -166,6 +166,33 @@ configuration_all:
     server: true
     client: false
     nuxtclient: false
+  # Identity Management
+  FEATURE_IDENTITY_MANAGEMENT_ENABLED:
+    value: 'false'
```

Manuel Linter says use " please
```suggestion
    value: "false"
```

---

done